### PR TITLE
Send anonymous user ID and first source URL to HubSpot with Cloud sign ups

### PR DESCRIPTION
**Pull request by attfarhan**
Closes https://github.com/sourcegraph/sourcegraph/issues/17939 and closes https://github.com/sourcegraph/sourcegraph/issues/17938. 

When a user signs up on Sourcegraph Cloud, we currently do not get any data about their activity prior to signup. 

This PR populates the Anonymous User ID field in HubSpot with the value from the `sourcegraphAnonymousUid` cookie when a user signs up, so we know if they have taken actions on Sourcegraph Cloud prior to signing up. It also populates the First Source URL field in HubSpot with the value from `sourcegraphSourceUrl` cookie so we know what the first ever URL a user visited across all Sourcegraph sites prior to signup.

### CODE REVIEW

File: cmd/frontend/internal/cli/serve_cmd.go [hunk 1]
```diff
@@ -123,7 +123,7 @@ func Main(enterpriseSetupHook func() enterprise.Services) error {
 	log.SetPrefix("")
 
 	if err := profiler.Init(); err != nil {
-		log.Fatalf("failed to initialize profiling: %v", err)
+		log.Printf("failed to initialize profiling: %v", err)
```
Comment (arussellsaw) [inline cmd/frontend/internal/cli/serve_cmd.go:126]: i don't think this diff is meant to be here 😅   same for the ones below

File: internal/hubspot/contacts.go [hunk 2]
```diff
@@ -37,9 +37,11 @@ func (c *Client) baseContactURL(email string) *url.URL {
...
 // ContactProperties represent HubSpot user properties
 type ContactProperties struct {
-	UserID        string `json:"user_id"`
-	IsServerAdmin bool   `json:"is_server_admin"`
-	LatestPing    int64  `json:"latest_ping"`
+	UserID          string `json:"user_id"`
+	IsServerAdmin   bool   `json:"is_server_admin"`
+	LatestPing      int64  `json:"latest_ping"`
+	AnonymousUserID string `json:"anonymous_user_id"`
+	FirstSourceUrl  string `json:"first_source_url"`
```
Comment (arussellsaw) [inline internal/hubspot/contacts.go:44]: i'm agreeing with the lint bot here, FirstSourceURL is the idomatic way to write this

**Comment by ebrodymoore** [comment]
I'll trust the more knowledgable people to review this PR but thanks again for doing this @attfarhan  😄

**Review by unknwon — COMMENTED** [review]
Agree with @arussellsaw's comments.

**Review by arussellsaw — APPROVED** [review]

**Review by unknwon — APPROVED** [review]

**Review by artemruts — APPROVED** [review]
FE LGTM

**Pull request merged**